### PR TITLE
Update tango examples to work with new API and match style of other examples

### PR DESCRIPTION
**Pull request by lincolnfrog**
Caveat - occlusion example shader isn't working properly to avoid clipping the teapot on the surface on which it sits

### CODE REVIEW

File: examples/marker.html [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+});
+
+
+function init() {
+  // Initialize the dat.GUI.
+  var datGUI = new dat.GUI();
+  gui = new GUI();
+  datGUI.add(gui, "showSeeThroughCamera").name("Show seethrough camera");
+  datGUI.add(gui, "markerTypeString", ["AR", "QRCODE"]).onFinishChange(function(value) {
+      console.log(value + ", " + this);
```
Comment (jsantell) [inline examples/marker.html]: nit: console log and indentation

File: examples/marker.html [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+  // the projection matrix provided from the device, so that the
+  // perspective camera's depth planes and field of view matches
+  // the physical camera on the device.
+  camera = new THREE.ARPerspectiveCamera(vrDisplay, 60, window.innerWidth / window.innerHeight, 0.01, 100);
+
+  // Create a model to be placed in the world using picking
+  model = new THREE.Mesh(new THREE.ConeBufferGeometry(
+    MODEL_SIZE_IN_METERS / 2, MODEL_SIZE_IN_METERS), 
+    new THREE.MeshLambertMaterial( {color: 0x888888 } ));
+  // model = new THREE.Mesh(new THREE.SphereBufferGeometry(
```
Comment (jsantell) [inline examples/marker.html]: nit: commented out code

File: examples/marker.html [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+ * projection matrix provided from the device
+ */
+function onWindowResize () {
+  camera.aspect = window.innerWidth / window.innerHeight;
+  camera.updateProjectionMatrix();
+  renderer.setSize(window.innerWidth, window.innerHeight);
+}
+
+function update() {
+    // Render the device's camera stream on screen first of all.
```
Comment (jsantell) [inline examples/marker.html]: nit: identation

File: examples/marker.html [hunk 4]
```diff
@@ -0,0 +1,236 @@
...
+  camera.updateProjectionMatrix();
+
+  // Update our perspective camera's positioning
+  vrControls.update();
+
+  // Render our three.js virtual scene
+  renderer.clearDepth();
+  renderer.render(scene, camera);
+
+  if (vrDisplay) {
```
Comment (jsantell) [inline examples/marker.html]: I don't think this conditional is necessary, we never enter the render loop if there's no valid AR device

File: examples/marker.html [hunk 5]
```diff
@@ -0,0 +1,236 @@
...
+    var markers = vrDisplay.getMarkers(gui.markerType, 0.1397);
```
Comment (jsantell) [inline examples/marker.html]: Magic number, what is `0.1397`?

File: examples/occlusion.html [hunk 6]
```diff
@@ -0,0 +1,392 @@
...
+    picking();
+  });
+  // Control the rotation of the model
+  var lastX = 0;
+  renderer.domElement.addEventListener("touchstart", function(event) {
+    if (event.changedTouches.length > 0)
+      lastX = event.changedTouches[0].pageX;
+  });
+  renderer.domElement.addEventListener("touchmove", function(event) {
+      if (event.changedTouches.length > 0 && vrDisplay) {
```
Comment (jsantell) [inline examples/occlusion.html]: nit: identation

File: examples/occlusion.html [hunk 7]
```diff
@@ -0,0 +1,392 @@
...
+        lastX = x;
+        model.rotateX(THREE.Math.degToRad(diffX));
+      }
+  });
+
+  update();
+}
+function picking() {
+  if (vrDisplay) {
+    var hits =
```
Comment (jsantell) [inline examples/occlusion.html]: nit: this statement should be in one line

File: examples/occlusion.html [hunk 8]
```diff
@@ -0,0 +1,392 @@
...
+        vrDisplay.hitTest(pos.x, pos.y);
+    if (hits && hits.length > 0) {
+      var hit = hits[0];
+      // Orient the cone model at the position and orientation of the hit.
```
Comment (jsantell) [inline examples/occlusion.html]: s/cone/teapot

File: examples/occlusion.html [hunk 9]
```diff
@@ -0,0 +1,392 @@
...
+      THREE.ARUtils.placeObjectAtHit(model, hit, 1, true);
+      points.material.uniforms.plane.value.fromArray(
+          [hit.modelMatrix[4], hit.modelMatrix[5], hit.modelMatrix[6], hit.modelMatrix[7]]);
+      /*
```
Comment (jsantell) [inline examples/occlusion.html]: This chunk of comments can probably be removed

File: examples/occlusion.html [hunk 10]
```diff
@@ -0,0 +1,392 @@
...
+    picking();
+  }
+
+  // Update the point cloud. Only if the point cloud will be shown the
+  // geometry is also updated.
+  pointCloud.update(gui.usePointCloudForDepth, 0, gui.transformPoints);
+
+  // Render the point cloud
+  if (this.gui.usePointCloudForDepth) {
+    this.renderer.context.colorMask(false, false, false, false);
```
Comment (jsantell) [inline examples/occlusion.html]: This is where the occlusion magic happens -- some extra comments here would be helpful as a standard example for occlusion in web AR

**Review by jsantell — APPROVED** [review]
Some comments and lint tests failing; looks good!

File: examples/marker.html (hunk 5 above)
Comment (lincolnfrog) [reply examples/marker.html]: size of marker. I'll make a constant.

File: examples/marker.html (hunk 4 above)
Comment (lincolnfrog) [reply examples/marker.html]: done

File: examples/marker.html (hunk 3 above)
Comment (lincolnfrog) [reply examples/marker.html]: done

File: examples/marker.html (hunk 2 above)
Comment (lincolnfrog) [reply examples/marker.html]: done

File: examples/marker.html (hunk 1 above)
Comment (lincolnfrog) [reply examples/marker.html]: done

File: examples/occlusion.html (hunk 6 above)
Comment (lincolnfrog) [reply examples/occlusion.html]: done

File: examples/occlusion.html (hunk 7 above)
Comment (lincolnfrog) [reply examples/occlusion.html]: done

File: examples/occlusion.html (hunk 8 above)
Comment (lincolnfrog) [reply examples/occlusion.html]: done

File: examples/occlusion.html (hunk 9 above)
Comment (lincolnfrog) [reply examples/occlusion.html]: done

File: examples/occlusion.html (hunk 10 above)
Comment (lincolnfrog) [reply examples/occlusion.html]: done

**Pull request merged**